cli/integration_tests/run_summary/errortask.t

```diff
@@ -0,0 +1,112 @@
+Setup
+  $ . ${TESTDIR}/../_helpers/setup.sh
+  $ . ${TESTDIR}/../_helpers/setup_monorepo.sh $(pwd)
+
+  $ rm -rf .turbo/runs
+
+Without --continue flag
+  $ ${TURBO} run maybefails --summarize --output-logs=hash-only > /dev/null 2>&1
+
+  $ source "$TESTDIR/../_helpers/run-summary-utils.sh"
+  $ SUMMARY=$(/bin/ls .turbo/runs/*.json | head -n1)
+
+  $ echo $(getSummaryTaskId $SUMMARY "my-app#maybefails") | jq
+  {
+    "taskId": "my-app#maybefails",
+    "task": "maybefails",
+    "package": "my-app",
+    "hash": "c11710f9379cb7ea",
+    "inputs": {
+      "package.json": "6bcf57fd6ff30d1a6f40ad8d8d08e8b940fc7e3b"
+    },
+    "hashOfExternalDependencies": "ccab0b28617f1f56",
+    "cache": {
+      "local": false,
+      "remote": false
+    },
+    "command": "exit 4",
+    "cliArguments": [],
+    "outputs": null,
+    "excludedOutputs": null,
+    "logFile": "apps/my-app/.turbo/turbo-maybefails.log",
+    "directory": "apps/my-app",
+    "dependencies": [],
+    "dependents": [],
+    "resolvedTaskDefinition": {
+      "outputs": [],
+      "cache": true,
+      "dependsOn": [],
+      "inputs": [],
+      "outputMode": "full",
+      "env": [],
+      "persistent": false
+    },
+    "expandedOutputs": [],
+    "framework": "",
+    "environmentVariables": {
+      "configured": [],
+      "inferred": [],
+      "global": [
+        "SOME_ENV_VAR=",
+        "VERCEL_ANALYTICS_ID="
+      ]
+    },
+    "execution": {
+      "startTime": [0-9]+, (re)
+      "endTime": [0-9]+, (re)
+      "error": {},
```

Comment:
this should probably be the error, not sure why it's empty (or maybe test fixture is wrong)